runtime/doc/options.txt

```diff
@@ -2219,10 +2219,10 @@ A jump table for the options with a short description can be found at |Q_op|.
 	viminfo file, etc.  It sets the kind of characters which Vim can work
 	with.  See |encoding-names| for the possible values.
 
-	NOTE: Changing this option will not change the encoding of the
-	existing text in Vim.  It may cause non-ASCII text to become invalid.
-	It should normally be kept at its default value, or set when Vim
-	starts up.  See |multibyte|.  To reload the menus see |:menutrans|.
+	This option cannot be changed after sourcing of |vimrc| files are done.
+	Otherwise it would have caused non-ASCII text inside Vim to become
```

Comment:
Following the discussion in #1250 we should update the GTK paragraph to a more general statement about GUI:s and remote plugins might be only fully functional when `encoding=utf-8` (at least not  "care has been taken to allow different values of
	'encoding'" )